GDJS/Runtime/gd.ts

```diff
@@ -558,8 +558,28 @@ namespace gdjs {
       return diff / Math.min(absA + absB, Number.MAX_VALUE) < epsilon;
     }
   };
+
+  const asynchronouslyLibraryPromises: Array<Promise<any>> = [];
+
+  /**
+   * Register a promise that a library will be loaded.
+   *
+   * This method must be called by any library that loads asynchronously.
+   */
+  export const registerAsynchronouslyLibraryPromise = (
```

Comment:
Should this be named `registerAsynchronouslyLoadedLibraryPromise` or `registerPromiseForLibraryLoading`. I had trouble reading the english of the variable at first